### PR TITLE
RC_Channel: Updated RC_Channel Example

### PR DESCRIPTION
**Pull request by hiro2233**
Updated RC_Channel example.

Prior code example, make floating point interrupt.

### CODE REVIEW

File: libraries/RC_Channel/examples/RC_Channel/RC_Channel.cpp [hunk 1]
```diff
@@ -26,8 +26,9 @@ static RC_Channel rc_5(CH_5);
 static RC_Channel rc_6(CH_6);
 static RC_Channel rc_7(CH_7);
 static RC_Channel rc_8(CH_8);
-static RC_Channel *rc = &rc_1;
```
Comment (tridge) [inline libraries/RC_Channel/examples/RC_Channel/RC_Channel.cpp]: this was deliberately an array so we can test all channels

File: libraries/RC_Channel/examples/RC_Channel/RC_Channel.cpp [hunk 2]
```diff
@@ -41,28 +42,30 @@ void setup()
...
 
     rc_5.set_range(0,1000);
-    rc_6.set_range(200,800);
+    rc_6.set_range(0,800);
 
     rc_7.set_range(0,1000);
 
     rc_8.set_range(0,1000);
     for (int i=0; i<NUM_CHANNELS; i++) {
-        rc[i].enable_out();
```
Comment (tridge) [inline libraries/RC_Channel/examples/RC_Channel/RC_Channel.cpp]: this breaks the point of the test, which is to test all channels

**Comment by tridge** [comment]
we should fix the exception, not remove the functionality of the test

**Comment by tridge** [comment]
i've pushed a fix for the bug. It was assuming channels were packed in memory

**Comment by hiro2233** [comment]
Nice fix!
It's working perfectly, thanks.

**Pull request closed without merging**